FAQ.md

```diff
@@ -2,68 +2,68 @@
 
-You can turn any of your existing Web Apps into a COBI module with a few extra lines of javascript.
+You can turn any of your existing web apps into a module with a few extra lines of javascript.
 
-#### 2. To test your COBI Module on a bike or eBike:
+#### 2. To test your Module on a bike or eBike:
 
-* Online hosting for your Module
-* iOS Device (minimum iPhone 5 with iOS 9.0)
-* [COBI System](https://get.cobi.bike)
-* Registration as a COBI Developer on [my.cobi.bike](https://my.cobi.bike)
+* Online hosting for your module
+* iOS Device (minimum iPhone 5 with iOS 10.0)
+* [COBI.bike system](https://get.cobi.bike)
+* Registration as a developer on [my.cobi.bike](https://my.cobi.bike)
 
-### Do I need to purchase a COBI system to start building a module?
+### Do I need to purchase a COBI.bike system to start building a module?
 
-No, you can get started for free with the [COBI DevKit Simulator](https://github.com/cobi-bike/COBI.js-simulator) and test on a COBI system when you're ready. If you don't have a COBI yet, you can purchase a Developer Edition on [cobi.bike/devkit](https://cobi.bike/devkit) or any other COBI System in our [online store](https://get.cobi.bike).
+No, you can get started for free with the [DevKit Simulator](https://github.com/cobi-bike/COBI.js-simulator) and test on a COBI.bike system when you're ready. If you don't have one yet, you can purchase a developer edition on [cobi.bike/devkit](https://cobi.bike/devkit) or any other COBI.bike system in our [online store](https://get.cobi.bike).
 
-### How can I turn my website / webapp into a COBI module?
+### How can I turn my website / webapp into a module?
 
-Include COBI.js in your html and start subscribing to riding data and remote control events.
+Include `COBI.js` in your html and start subscribing to riding data and remote control events.
 
-### Can you recommend any additional development tools / platforms to build COBI modules?
+### Can you recommend any additional development tools / platforms to build modules?
 
-We like getting inspired on [codepen.io](https://codepen.io) and building prototypes on [glitch.me](https://glitch.me). Both work great with the [COBI DevKit Simulator](https://github.com/cobi-bike/COBI.js-simulator).
+We like getting inspired on [codepen.io](https://codepen.io) and building prototypes on [glitch.me](https://glitch.me). Both work great with the [DevKit Simulator](https://github.com/cobi-bike/COBI.js-simulator).
 
 ## Technical FAQ
 
-### I don't know how to test my module within the COBI app
+### I don't know how to test my module within the COBI.bike app
 
-Did you already sign up as a COBI Developer on [my.cobi.bike](https://my.cobi.bike)? If yes, please follow the instructions on the [DevKit site](https://github.com/cobi-bike/COBI-DevKit).
+Did you already sign up as a developer on [my.cobi.bike](https://my.cobi.bike)? If yes, please follow the instructions on the [DevKit site](https://github.com/cobi-bike/COBI-DevKit).
```

Comment:
the `Devkit site` link shouldnt be trusted. AFAIK github keeps the repo renaming for 6 months or so, so I would suggest that we change it now rather than later when it breaks. 

PS: I am talking about the `cobi-bike/COBI-DevKit` url. It should be `cobi-bike/DevKit`

Reply:
Good point. There may be other outdated GH links. Can you check the repositories for such links?

Reply:
I am checking all PR that github shows me in the notifications. I will let you know if I find something else

Reply:
I guess this is not really related to the current set of PRs because the repo names were changed before already.